docs/ZABBIX_PROXY_ROLE.md

```diff
@@ -224,7 +216,7 @@ We need to have the following dependencies met:
 
 ```yaml
 zabbix_proxy_database: mysql
-zabbix_proxy_database_long: mysql
+zabbix_db_type_long: mysql
```

Comment:
I may be mistaken, but does this deviate official ansible best practices where role defaults should be prefixed with role name? 

Not a suggestion, just an invitation for a discussion.

Reply:
Makes sense.